src/style.js

```diff
@@ -92,6 +92,18 @@ export function getVendorJsPropertyName(style, camelCase, opt_bypassCache) {
 
 
 /**
+ * Sets the CSS styles of the specified element with !important. The styles
+ * are specified as a map from CSS property names to their values.
+ * @param {!Element} element
+ * @param {!Object<string, *>} styles
+ */
+export function setImportantStyles(element, styles) {
+  element.style.cssText +=
```

Comment:
Ah, ok. Looking into it, you can pass `"important"` as the third argument to `CSSStyleDeclaration#setProperty`, which should avoid a large amount of the parsing this requires. Mind switching to that?